### DISCUSSION TITLE
Sprint Retro Feedback

### DISCUSSION THREAD

**Discussion by Coffee2Bits** [DevOps-MBSE/AaC#83] [Archive]
Since we're incorporating some of the Agile Scrum ceremonies we should make an effort to use document some of the feedback that we'd expect from a retrospective. For now, I'll document it in this discussion until we find another, more preferable medium to have this discussion.

Sprints are listed in descending order so that the latest sprint is at the top.

### Sprint 2
10/25/21 - 11/5/21

What went well in the Sprint:
- [x] Feature development 
- [x] Collaboration (Discussions and pairing) 

What could be improved:
- [ ] Github issues that are concerned with generation should include amble descriptions about what the output should contain and look like,
- [ ] Unexpected work that's not in the scope of a story and is more than a few minor changes should be broken out into another, smaller PR
- [ ] Increase the use of draft PRs

What will we commit to improving in the next Sprint:
- [ ] Increase the use of draft PRs
- [ ] Smaller PRs
- [ ] Plugin testing strategy 

### Sprint 1
10/11/21 - 10/22/21

What went well in the Sprint:
What could be improved:
What will we commit to improving in the next Sprint:

**Comment by Coffee2Bits** [comment]
I went ahead and filled in an improvement item for Sprint 2 based on a conversation that Cameron and I had, but please feel free to post suggestions for Sprint 1 and/or 2, especially since we don't have a set retrospective meeting.

**Comment by jondavid-black** [comment]
Have you ever used the wiki built-in to GitHub?  I haven't, but if it gets us anything Confluence-like it may be viable to document retro content.  It may be worth an experiment.

**Reply by Coffee2Bits** [reply]
I haven't either, but that's a good idea. I'm a little worried about using the wiki to host team retrospectives as that might be misleading to people who are expecting a wiki to contain wiki information. However, we do have the GitHub pages and the readme. So, if we're not worried about potential confusion, then the wiki will probably be the more manageable solution long term.

**Comment by jondavid-black** [comment]
For me, sprint 2 was a signal that you guys can't rely on me to get issues worked.  I think we should make sure I'm not a roadblock going forward.  I'll find time to finish up my current issue but will stop taking on work until things settle down on other fronts.  The good news is we have at least 1 and maybe 2 others who are interested in contributing.

**Comment by Coffee2Bits** [comment]
Closing out this old discussion.

**Comment by lizzcondrey** [comment]
Closing since this is marked as resolved and the sprint is past.
